bin/run_functions.js

```diff
@@ -406,7 +419,7 @@ function executeProject(filename, project, options) {
                         callback();
                     });
             },
-        function (callback) {
+            function (callback) {
```

Comment:
Same as above ^

Reply:
Both indentation inconsistencies should now be corrected.